market/connector/currencyConverter/product.json

```diff
@@ -0,0 +1,58 @@
+{
+    "properties": {
+        "openApiUrl": "openapi.json"
+    },
+    "installers": [
+        {
+            "id": "rest-client",
+            "data": {
+                "openApiUrl": "@openApiUrl",
```

Comment:
I think with the version we have a slightly different than our ${version} ... if we sould have an end separator it's easier to parse I think